Error when uploading a library

Using Add-UiPathLibrary  throws an error 
"2020-01-13T09:18:01.6236856Z ##[error]Operation returned an invalid status code '207'"

Local execution with verbose enabled generated the log bellow. It was executed against current version of platform.uipath.com 

VERBOSE: Error: $RuntimeException:$Cannot parse response extracting error message from
{"@odata.context":"https://platform.uipath.com/odata/$metadata#Collection(UiPath.Orchestrator.Application.Dto.BulkItemD
to_1OfString)","value":[{"Key":"R:\\temp\\l\\LibraryTest.1.0.1.nupkg","Status":"OK","Body":"{\"Id\":\"LibraryTest\",\"V
ersion\":\"1.0.1\"}"}]}
Add-UiPathLibrary : Operation returned an invalid status code '207'
At line:1 char:1
+ Add-UiPathLibrary -LibraryPackage R:\temp\l\LibraryTest.1.0.1.nupkg - ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Add-UiPathLibrary], HttpOperationException
    + FullyQualifiedErrorId : Microsoft.Rest.HttpOperationException,UiPath.PowerShell.Cmdlets.AddLibrary

--- a/UiPath.PowerShell/Cmdlets/AddLibrary.cs
+++ b/UiPath.PowerShell/Cmdlets/AddLibrary.cs
@@ -2,6 +2,7 @@
 using System.Management.Automation;
 using UiPath.PowerShell.Util;
 using UiPath.Web.Client20183;
+using UiPath.Web.Client201910;
 
 namespace UiPath.PowerShell.Cmdlets
 {
@@ -15,7 +16,14 @@ namespace UiPath.PowerShell.Cmdlets
         {
             using (var fileStream = File.OpenRead(LibraryPackage))
             {
-                HandleHttpOperationException(() => Api_18_3.Libraries.UploadPackage(fileStream));
+                if (Supports(OrchestratorProtocolVersion.V19_10))
+                {
+                    HandleHttpOperationException(() => Api_19_10.Libraries.UploadPackage(fileStream));
+                }
+                else
+                {
+                    HandleHttpOperationException(() => Api_18_3.Libraries.UploadPackage(fileStream));
+                }
             }
         }
     }

--- a/UiPath.PowerShell/Util/OrchestratorProtocolVersion.cs
+++ b/UiPath.PowerShell/Util/OrchestratorProtocolVersion.cs
@@ -11,5 +11,7 @@ namespace UiPath.PowerShell.Util
         internal static  Version V18_3 => Version.Parse("6.0");
 
         internal static Version V18_4 => Version.Parse("7.0");
+
+        internal static Version V19_10 => Version.Parse("9.0");
     }
 }

